font-codegen/src/parsing.rs

```diff
@@ -275,8 +284,11 @@ pub(crate) enum FieldType {
     Scalar {
         typ: syn::Ident,
     },
-    Other {
-        typ: syn::Path,
+    Struct {
+        typ: syn::Ident,
+    },
+    PendingResolution {
```

Comment:
Might be useful to add a note here, something like:


```suggestion
    /// A type that may be a struct or a scalar.
    ///
    /// This only exists at parse time; when parsing is finished this will be 
    /// resolved (or be an error).
    PendingResolution {
```